source/_integrations/tami4.markdown

```diff
@@ -0,0 +1,18 @@
+---
+title: "Tami4 Edge / Edge+"
+description: "Tami4 Edge Integration"
+ha_release: "2022.11"
+ha_category: Other
+ha_iot_class: "Cloud Polling"
+ha_quality_scale: silver
+ha_config_flow: true
+ha_codeowners:
+  - '@Guy293'
+ha_domain: tami4
+---
+
+Support for Tami4 Edge / Edge+ by Strauss in Home Assistant.
+
+You can boil water and get filter / uv information, such as when you need to replace them.
+
+You can easily pair your device by confirming your phone number which is registered with your Edge app account.
```

Comment:
```suggestion
You can easily pair your device by confirming your phone number which is registered with your Edge app account.

{% include integrations/config_flow.md %}
```
It misses the "how to configure" section + I'm not sure this sentence really help to configure the service into Home Assistant.

Either:
- create a section before the HA configuration to explain how to configure your Tami4 account
- explain only the HA configuration part